integrations/docker/images/chip-build/Dockerfile

```diff
@@ -64,7 +64,8 @@ RUN set -x \
     && : # last line
 
 RUN set -x \
-    && pip3 install circleci attrs coloredlogs PyGithub pygit \
+    && pip3 install circleci attrs coloredlogs PyGithub pygit future \
+    portpicker mobly                                                 \
```

Comment:
do we need to lockdown mobly version in case it breaks in latest version?

Reply:
It won't install the latest version, it will install the version in Ubuntu 20.04 LTS which I wouldn't expect to receive feature updates.

If it breaks, it would most likely be when we try to upgrade the distribution used by the chip-build docker images.